ci-operator/step-registry/cucushift/installer/rehearse/aws/usgov/ipi/custom-dns/cucushift-installer-rehearse-aws-usgov-ipi-custom-dns-workflow.yaml

```diff
@@ -0,0 +1,10 @@
+workflow:
+  as: cucushift-installer-rehearse-aws-usgov-ipi-custom-dns
+  steps: 
+    pre:
+    - chain: cucushift-installer-rehearse-aws-ipi-custom-dns-provision
```

Comment:
Per https://github.com/openshift/release/pull/67158, do you mind to append `- ref: cucushift-installer-reportportal-marker` here ?